modules/store/schematics/ng-add/index.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  Rule,
+  SchematicContext,
+  Tree,
+  chain,
+  externalSchematic,
+} from '@angular-devkit/schematics';
+import { Schema as NgAddOptions } from './schema';
+
+export default function(options: NgAddOptions): Rule {
+  return (host: Tree, context: SchematicContext) => {
+    return chain([
+      externalSchematic('@ngrx/schematics', 'store', {
```

Comment:
This assumes that they already have `@ngrx/schematics` installed before they run `ng add @ngrx/store`. They should be able to add it without any previous dependencies.

Reply:
I see 2 possible solutions:
1. Use `ngrx/schematics` code for `ng-add`s and configure tsconfig to bundle dependencies + copy template files on build step
2. Implement separate schematics for all of `ng-add`s

IMHO better to go 1st way to have schematics code re-used, but want to know your opinions.

Reply:
I'm going to land a PR soon that helps with this. We're going to move the schematics utility functions into `modules/schematics-core`. This way we can share them across schematics packages for each lib that needs them.